apps/pre-award/copilot/environments/addons/communities-redirect-cloudfront-function.yml

```diff
@@ -0,0 +1,72 @@
+Parameters:
+  App:
+    Type: String
+    Description: Your application's name.
+  Env:
+    Type: String
+    Description: The environment name your service, job, or workflow is being deployed to.
+
+Mappings:
+  DomainSuffixMap:
+    dev:
+      "DomainSuffix": ".dev.access-funding.communities.gov.uk"
+    test:
+      "DomainSuffix": ".test.access-funding.communities.gov.uk"
+    uat:
+      "DomainSuffix": ".uat.access-funding.communities.gov.uk."
```

Comment:
Sneaky extra period at the end?

Reply:
For levellingup we're on `.access-funding.test.levellingup.gov.uk` - has that `.test.` middle bit disappeared on the communities equivalent? I guess so if you've tested this and it works (nice!)

Reply:
@samuelhwilliams yes, good spot on that end period.

Yes, that is deliberate re: URLs. Plan is to scope cookies to e.g. `.test.access-funding.communities.gov.uk` so shouldn't leak to prod as long as this is done correctly?

Reply:
Ah, I see what you mean the prod cookie would be available on all subdomains...